tests/unit/acquisition/function/test_active_learning.py

```diff
@@ -392,3 +400,86 @@ def test_integrated_variance_reduction_builder_updates_without_retracing() -> No
 
     npt.assert_array_almost_equal(acq_fn(query_at), expected)
     assert acq_fn.__call__._get_tracing_count() == 1  # type: ignore
+
+
+def test_bayesian_active_learning_by_disagreement_builder_builds_acquisition_function() -> None:
+    x = to_default_float(tf.zeros([1, 1]))
+    y = to_default_float(tf.zeros([1, 1]))
+    model = VariationalGaussianProcess(vgp_model_bernoulli(x, y))
+    acq_fn = BayesianActiveLearningByDisagreement().prepare_acquisition_function(model)
+    query_at = tf.linspace([[-10]], [[10]], 100)
+    expected = bayesian_active_learning_by_disagreement(model, DEFAULTS.JITTER)(query_at)
+    npt.assert_array_almost_equal(acq_fn(query_at), expected)
+
+
+def test_bayesian_active_learning_by_disagreement_raise_on_non_bernoulli_vgp() -> None:
```

Comment:
we should have a test for non vgp model as well, no? you are checking for that type as well...

Reply:
yes, I think that also needed.

Reply:
by the way, we could also have an SVGP based classifier